website/docs/r/event_orchestration_integration.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: 'pagerduty'
+page_title: 'PagerDuty: pagerduty_event_orchestration_integration'
+sidebar_current: 'docs-pagerduty-resource-event-orchestration-integration'
+description: |-
+  Creates and manages an Integration for an Event Orchestration.
+---
+
+# pagerduty_event_orchestration_integration
+
+An Event Orchestration Integration allows you to create and manage multiple Integrations (and Routing Keys) per Event Orchestration _and_ will alow you to move (migrate) Integrations _between_ two Event Orchestrations.
+
+## Example of configuring an Integration for an Event Orchestration
+
+This example shows creating `Event Orchestration` and `Team` resources followed by creating an Event Orchestration Integration to handle Events sent to that Event Orchestration.
+
+-> When a new Event Orchestration is created there will be one Integration (and Routing Key) included by default. Example below shows how to create an extra Integration associated with this Event Orchestration.
```

Comment:
**Annotation:** Reminder for a user about included default integration on a new EO creation